lib/kitchen/directions/bake_numbered_table/v1.rb

```diff
@@ -3,20 +3,20 @@
 module Kitchen::Directions::BakeNumberedTable
   class V1
 
-    def bake(table:, number:, always_caption: false)
+    def bake(table:, number:, always_caption: false, title_css: "span[data-type='title']")
       Kitchen::Directions::BakeTableBody.v1(table: table, number: number)
 
       # TODO: extra spaces added here to match legacy implementation, but probably not meaningful?
       new_caption = ''
       caption_title = ''
 
-      if (title = table.first("span[data-type='title']")&.cut)
+      if (title = table.first(title_css)&.cut)
```

Comment:
I really like that you've thought of a way to keep the selector specific (& therefore fast!). What do you think of instead defining a `TableElement#title` or `TableElement#caption_title` method? This would allow the selector to be updated/added to easily, for example, to `'span[data-type="title"], h4[data-type="title"]'` (I think that's `or`-syntax for a css selector, but I'd have to double-check). I believe `CompositePageElement#title` does something similar.

Reply:
Actually, I think I got the root cause of this wrong-- the assembled uses all span titles.... but there's something along the way that turns some of them into h4s.